jupyter_client/session.py

```diff
@@ -43,6 +43,13 @@
     # limiting the surface of attack
     def compare_digest(a,b): return a == b
 
+try:
+    from datetime import timezone
+    utc = timezone.utc
+except ImportError:
+    from dateutil.tz import tzutc
```

Comment:
Let's have a comment here noting that it's a Python 3/2 difference, for clarity.